### ISSUE TITLE
Angular translate causes false positives on "[A11y] Missing ARIA label on input"

### ISSUE THREAD

**Issue by fynnfeldpausch** [haiilo/catalyst#455]

**Comment by fynnfeldpausch** [comment]
due to async pipe
